core/sql/regress/udr/EXPECTED001

```diff
@@ -195,6 +244,33 @@ CREATE TABLE_MAPPING FUNCTION TRAFODION.SCH.SESSIONIZE_DYNAMIC
 
 --- SQL operation complete.
 >>
+>>get table_mapping functions for library TEST001_Java ;
```

Comment:
I found that difficult to do in our parser without increasing conflicts. This is not to say that we should not try again.